### PR TITLE
Add ActivityPub

### PR DESCRIPTION
**Pull request by mlinksva**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [x] Curating a new Topic page

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

Not 100% certain it should be. Only [21 repos](https://github.com/topics/activitypub) but fairly new/potentially important -- [became a W3C recommendation Jan 23](https://www.w3.org/blog/news/archives/6785), developed on GitHub, as is the leading implementation (Mastodon), by far the most successful decentralized social networking project so far. It has a nice (and public domain) logo. :smile:

### CODE REVIEW

**Comment by cheshire137** [comment]
> Not 100% certain it should be.

I think it's fine to add context around topics that don't have a lot of repositories yet! We don't have to feature them on the main [Topics page](https://github.com/topics), but if a user lands on the topic page for a less-used topic, it's still nice to have context explaining that topic.

File: topics/activitypub/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+display_name: ActivityPub
+github_url: https://github.com/w3c/activitypub
+logo: activitypub.png
+related: decentralized, federated, protocol, social-network, w3c
+short_description: Decentralized social networking protocol.
+topic: activitypub
+url: https://activitypub.rocks/
```
Comment (cheshire137) [inline topics/activitypub/index.md]: I see on the w3c/activitypub repo, the web site is listed as http://w3c.github.io/activitypub/. However, I see some of the links on http://w3c.github.io/activitypub/ point to the activitypub.rocks domain, so I think this `url:` is fine as the activitypub.rocks site seems more user-friendly and provides a good introduction.

File: topics/activitypub/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+display_name: ActivityPub
+github_url: https://github.com/w3c/activitypub
+logo: activitypub.png
+related: decentralized, federated, protocol, social-network, w3c
+short_description: Decentralized social networking protocol.
```
Comment (cheshire137) [inline topics/activitypub/index.md]: Can you rephrase to a full sentence? Maybe "ActivityPub is an open, decentralized social networking protocol."

File: topics/activitypub/index.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+topic: activitypub
+url: https://activitypub.rocks/
+wikipedia_url: https://en.wikipedia.org/wiki/ActivityPub
+---
+ActivityPub is a decentralized social networking protocol standardized by the W3C. It includes both an API for social networking clients and an API for server-server federation.
```
Comment (cheshire137) [inline topics/activitypub/index.md]: Could spell out W3C as "World Wide Web Consortium (W3C)".

File: topics/activitypub/index.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+display_name: ActivityPub
+github_url: https://github.com/w3c/activitypub
+logo: activitypub.png
+related: decentralized, federated, protocol, social-network, w3c
```
Comment (cheshire137) [inline topics/activitypub/index.md]: Could add `api` for users who see API mentioned in the description and want to know more about that.

**Comment by MikeMcQuaid** [comment]
Looks good to me but will let @cheshire137 have the final say!
